include/boost/sml.hpp

```diff
@@ -1703,6 +1703,23 @@ class sm {
     });
     return result;
   }
+  template <class T = aux::identity<sm_t>, class TState>
+  bool is_one_of_current_states_as(const TState&) const {
```

Comment:
sml.hpp should not be changed directly as it's a generated file using tools/pph.sh